src/main/java/propertyFiles/Bundle.properties

```diff
@@ -874,23 +874,23 @@ dataverse.publish.header=Publish Dataverse
 dataverse.nopublished=No Published Dataverses
 dataverse.nopublished.tip=In order to use this feature you must have at least one published dataverse.
 dataverse.contact=Email Dataverse Contact
-dataverse.link=Link Dataverse
-dataverse.link.btn.tip=Link to Your Dataverse
-dataverse.link.yourDataverses=Your Dataverse
-dataverse.link.yourDataverses.inputPlaceholder=Enter Dataverse Name
-dataverse.link.save=Save Linked Dataverse
-dataverse.link.dataverse.choose=Choose which of your dataverses you would like to link this dataverse to.
-dataverse.link.dataset.choose=Enter the name of the dataverse you would like to link this dataset to. If you need to remove this link in the future, please contact {0}.
-dataverse.link.dataset.none=No linkable dataverses available.
-dataverse.link.no.choice=You have one dataverse you can add linked dataverses and datasets in.
-dataverse.link.no.linkable=To be able to link a dataverse or dataset, you need to have your own dataverse. Create a dataverse to get started.
-dataverse.link.no.linkable.remaining=You have already linked all of your eligible dataverses.
+dataverse.link=Link Collection
+dataverse.link.btn.tip=Link to Your Collection
+dataverse.link.yourDataverses=Your Collection
```

Comment:
typo? (I noticed the before was also singular, so I'm not sure of the contect, but the key is plural)

Reply:
It was singular in the original. I'll take a look at it in context and maybe change the key name.